classes/PlayerHandler.js

```diff
@@ -70,11 +70,11 @@ module.exports = class PlayerHandler {
 	 * Sends a message to the bound text channel.
 	 * @param {string} msg The message to be used.
 	 * @param {{title?: string, footer?: string, thumbnail?: string, additionalEmbeds?: EmbedBuilder[], files?: import('discord.js').FileOptions[], components?: import('discord.js').ActionRowBuilder[]}} [embedExtras] Extra data to be passed to the embed.
-	 * @param {boolean} [error] Whether or not the message is an error.
+	 * @param {"success"|"neutral"|"warning"|"error"} [type=neutral] Type of the message. Defaults to success.
 	 * @returns {Promise<import('discord.js').Message>|Promise<import('discord-api-types/v10').APIMessage>|boolean} The message that was sent.
 	 */
-	async send(msg, embedExtras, error) {
-		const sendData = this.sendDataConstructor(msg, embedExtras, error);
+	async send(msg, embedExtras, type) {
```

Comment:
"Defaults to success", this method's `type` parameter has no default value.

Remove doc @param `[type=neutral]`, this method's parameter `type` has no default value.